Library/Homebrew/formula_installer.rb

```diff
@@ -589,7 +589,7 @@ def build
       -I #{HOMEBREW_LOAD_PATH}
       --
       #{HOMEBREW_LIBRARY_PATH}/build.rb
-      #{formula.path}
+      #{formula.alias_path || formula.path}
```

Comment:
I thought about this. My reasoning for leaving it as-is is that having a method called `alias_path_or_path` wouldn't really be reducing repetition at all. It just occurred to me that `specified_path` would work as a name, though, so I'll move it into a method.